src/Server.h

```diff
@@ -214,6 +229,18 @@ class cServer
 	int m_MaxPlayers;
 	bool m_bIsHardcore;
 
+	// TODO: std::map<int, cForgeMods &> data structure?
```

Comment:
Removed the separate member variables in favor of a map, but not clear on how I can get the list of supported protocols? I see there is an unnamed enum in cProtocolRecognizer, and GetVersionTextFromInt() switch/cases on each version, and there are string representations `MCS_CLIENT_VERSIONS` and `MCS_PROTOCOL_VERSIONS`, but couldn't find a method to get an array/list/vector of the supported protocols. For now I added an array I iterate over to initialize (must be done at initialization time, so that RegisterForgeMod() adds to all supported protocols by default) in cServer, but should this array be moved to cProtocolRecognizer?